geocity/apps/submissions/views.py

```diff
@@ -954,6 +954,20 @@ def archive(self):
                     submission = get_submission_for_user_or_404(
                         self.request.user, submission_id
                     )
+
+                    # Prevent from archiving a submission where user isn't pilot.
+                    # User could be pilot for other submissions and validator for the one he's trying to archive
+                    if not permissions.is_backoffice_of_submission(
+                        self.request.user, submission
+                    ):
+                        return JsonResponse(
+                            data={
+                                "error": True,
+                                "message": self.permission_error_message,
+                            },
+                            status=403,
+                        )
+
```

Comment:
Explain which submission could not be archived to the user ? If many submissions are selected for archiving ans only one is causing the problem just list which could not be archived ? It's a nice to have, not a must.

Reply:
@monodo is that good like that ? Should we use `id` instead of `n°`
![image](https://github.com/yverdon/geocity/assets/36194606/b6d38fdd-d3b0-49fc-8afe-3c518d52bb00)

Trying to not add another loop just to list all submissions the user can't archive, but we can at least tell the one throwing an error